Scripts/Simulation_scripts/Energy_System.py

```diff
@@ -359,7 +386,7 @@ def get_storage_profile(self, start_year = 0, end_year = 4, PV_size = 10, **opti
         
 #   Initialise power generation, including degradation of PV
         PV_generation = PV_size * pd.DataFrame(self.get_PV_generation()[start_hour:end_hour].values 
-                                     * Solar().solar_degradation()[0:(end_hour-start_hour)].values)
+                                     * Solar(**self.kwargs).solar_degradation()[0:(end_hour-start_hour)].values)
```

Comment:
Not best sticking `kwargs` on the class as it's unclear what's stored in it and what's being passed through 🐱